setup.py

```diff
@@ -53,6 +53,10 @@ def get_tests_require() -> List[str]:
 def get_extras_require() -> Dict[str, List[str]]:
 
     requirements = {
+        "benchmark": [
```

Comment:
`virtualenv` is removed since it is builtin.

https://github.com/optuna/optuna/pull/3517/files#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7L174-L177